home_assistant_datasets/tool/assist/collect.py

```diff
@@ -144,6 +154,7 @@ def run(args: argparse.Namespace) -> int:
         f"--dataset={args.dataset or ''}",
         f"--model_output_dir={args.model_output_dir or ''}",
         f"--categories={args.categories or ''}",
+        f"--count={args.count}",
```

Comment:
I believe this changes the logic here: https://github.com/allenporter/home-assistant-datasets/blob/1affac26612ede7b237dc30c50d64fd8bc6031db/home_assistant_datasets/tool/data_model.py#L175 given count is always non-none now.

(I haven't thought about if we should update all CIs for all evals, so wanted to be intentional about enabling it)

Want to remove the default value and add the arg if its present?

Reply:
What i mean is don't set this flag ig its not set